src/LineChart.ts

```diff
@@ -35,11 +35,15 @@ module n3Charts {
       factoryMgr.registerMany([
         ['container', Factory.Container, element[0]],
         ['tooltip', Factory.Tooltip, element[0]],
+        ['legend', Factory.Legend, element[0]],
         ['transitions', Factory.Transition],
         ['x-axis', Factory.Axis, Factory.Axis.SIDE_X],
         ['y-axis', Factory.Axis, Factory.Axis.SIDE_Y],
-        ['series-column', Factory.Series.Column],
+
+        // This order is important, otherwise it can mess up with the tooltip
+        // (and you don't want to mess up with a tooltip, trust me).
```

Comment:
alright, this makes sense - maybe we could add this explanation into the comment